released/packages/coq-fcsl-pcm/coq-fcsl-pcm.1.5.1/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "fcsl@software.imdea.org"
+version: "dev"
+
+homepage: "https://github.com/imdea-software/fcsl-pcm"
+dev-repo: "git+https://github.com/imdea-software/fcsl-pcm.git"
+bug-reports: "https://github.com/imdea-software/fcsl-pcm/issues"
+license: "Apache-2.0"
+
+synopsis: "Partial Commutative Monoids"
```

Comment:
```suggestion
synopsis: "Coq library of Partial Commutative Monoids"
```
Maybe a bit more context here?

Reply:
Has been this way since 1.0.0 but sure, why not